Gemfile.jruby-1.9.lock.release

```diff
@@ -746,3 +747,4 @@ DEPENDENCIES
   stud (~> 0.0.22)
   term-ansicolor (~> 1.3.2)
   tins (= 1.6)
+  webrick (~> 1.3.1)
```

Comment:
it seems we are installing webrick because of https://github.com/elastic/logstash/pull/8850
We don't seem to need this in production, the Gemfile.template entry for webrick should be tagged as `:group => :development`
We can either fix it first in the gemfile or just manually leave these changes out in the PR

Reply:
@jsvd I have added the `:group => :development` in this PR, but `webrick` still ends up in the dependency graph (similar to how gems like `rspec`, `rack-test`, `docker-api`, `term-ansicolor`, etc., also tagged in that manner, are already in the dependency graph).